messages/proto/messages.proto

```diff
@@ -75,13 +75,20 @@ message CommitMessage {
 message RoundChangeMessage {
   // lastPreparedProposedBlock is the last block
   // to reach Q(N) - 1 PREPARE messages
-  bytes lastPreparedProposedBlock = 1;
+  ProposedBlock last_prepared_proposed_block = 1;
```

Comment:
Even though the [proto convention is to use snake case](https://protobuf.dev/programming-guides/style/#message-field-names), we utilize camel case in our proto declarations.
Either make everything snake case or everything camel case

Reply:
Changed back

[5f6e542](https://github.com/madz-lab/go-ibft/pull/10/commits/5f6e5424a90ff8e70adb209b4f9ad84572a6d00a)